easybuild/easyconfigs/d/DB/DB-6.2.32-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,26 @@
+name = 'DB'
+version = '6.2.32'
+
+homepage = 'http://www.oracle.com/technetwork/products/berkeleydb'
+
+description = """
+ Berkeley DB enables the development of custom data management solutions,
+ without the overhead traditionally associated with such custom projects.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+# download via http://www.oracle.com/technetwork/products/berkeleydb/downloads/, requires registration
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['a9c5e2b004a5777aa03510cfe5cd766a4a3b777713406b02809c17c8e0e7a8fb']
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+sanity_check_paths = {
+    'files': ["include/db.h"],
+    'dirs': [],
+}
```

Comment:
Why not using the most recent sanity check available in:

https://github.com/easybuilders/easybuild-easyconfigs/blob/b1332a8e969b5f379bb1e613b4e378b7a98d1a3e/easybuild/easyconfigs/d/DB/DB-6.2.32-intel-2017a.eb